packages/test-base/src/androidTest/kotlin/io/realm/kotlin/test/shared/ImportTests.kt

```diff
@@ -83,7 +83,7 @@ class ImportTests {
                 Float::class -> assertEquals(3.14f, managed.floatField)
                 Double::class -> assertEquals(1.19840122, managed.doubleField)
                 RealmInstant::class -> assertEquals(RealmInstant.from(100, 1000), managed.timestampField)
-                ObjectId::class -> assertEquals(ObjectId.from("507f1f77bcf86cd799439011"), managed.objectIdField)
+                ObjectId::class -> assertEquals(ObjectId.from("507f1f77bcf86cd799439011"), managed.realmObjectIdField)
```

Comment:
Don't we need to keep tests for both kbson objectID and our own?

Reply:
Yes, I marked it as reviewable too soon there were some missing tests + support on the config missing, and the changelog entry 🫣